src/sphinx_llm/tests/test_txt.py

```
@@ -162,6 +162,26 @@ def test_llms_txt_sitemap_links_exist(sphinx_build):
+    """Test that pages in llms.txt follow the Sphinx toctree order."""
+    _, build_dir, _ = sphinx_build
+
+    content = (build_dir / "llms.txt").read_text(encoding="utf-8")
+    page_titles = [
+        match.group(1)
+        for line in content.splitlines()
+        if (match := re.match(r"^- \[([^]]+)]\(", line))
+    ]
+
+    assert page_titles == [
+        "Welcome to sphinx-llm’s documentation!",
```

Ruff (0.16.0)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)